homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="Style/login.css">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+        <title>Login</title>
+    </head>
+    <body class="login__body">
+        <header class="about-me__header">
+            <img class="about-me__logo" alt="Logo" src="Pictures/logo.png">
+            <div class="about-me__header__menu">
+                <a class="header__link" href="./games.html" title="Games">Games</a>
+            </div>
+            <div class="about-me__header__menu">
+                <a class="header__link" href="./index.html" title="About">About</a>
+            </div>
+            <div class="about-me__header__menu">
+                <a class="header__link" href="./login.html" title="Login">Login</a>
+            </div>
+        </header>
+        <div class="login">
+            <h1 class="login__h1">Login page</h1>
+            <form class="login__form" action="./success.html">
+                <label class="login__fields">Email
+                </label>
+                <br>
+                <input class="login__input" type="email" name="auth_email" placeholder="Enter your email address" required>
+                <br>
+                <label class="login__fields">Password
+                </label>
+                <br>
+                <input class="login__input" type="password" name="auth_pass" placeholder="Enter your email address" required>
+                <br>
+                <label>
+                    <input type="checkbox" name="remember" required> I'm not a robot
+                </label>
+                <br>
+                <button class="login__button" type="submit" name="Login">Login </button>
```

Comment:
Please correct the border of the button
Also, your width of the input is fixed to 575px. But by [default](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing#content-box), the width of the border is added to this width, and as a result, you have 577px. So the inputs are wider than the button. 
Please fix the width or correct `box-sizing` to [border-box](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing#border-box)

 ![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/cbcb5ef3-573c-4384-8559-d542b5ba080e)

Reply:
fixed in 1ffea05f871d93574677456cc36ef53bd6c48564